docs/package.json

```diff
@@ -33,5 +33,6 @@
   "license": "MIT",
   "dependencies": {
     "fs-extra": "^8.0.0"
-  }
+  },
+  "private": true
```

Comment:
wouldn't the config be
```json
"publishConfig": {
    "access": "private"
  }
```
?

Reply:
No, it will mark the `@loopback/*` as a private npm package.

Reply:
ah I see, thank you for the explanation.